CHANGELOG.anychart.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+All notable changes to this widget will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Fixed
+- Each base chart component now has a `shouldComponentUpdate` method that compares the new props with the previous props. If there is a difference, it re-renders. This fixes issues arising due to re-plotting the Plotly Chart unnecessarily, such as when grouped legends are configured.
+- In MX 9.1 certain public client methods were removed. A check is now in-place to check the function's existence before use, else it fallsback to an alternative approach. This means ensures compatibility with MX 7, 8 and 9.
```

Comment:
I'm not quite sure whether it's useful for mendix developer to be so detailed about these fixes in the public release notes. E.g. for the first fix, I think the only thing that's really meaningful to them is that "issues with charts has been fixed in certain scenarios, like when grouped legends are configured".

Reply:
Better to explain in a more generic and user friendly way

Reply:
I agree, thanks!

Reply:
Changed.